src/apps/components/AppUrlLink/AppUrlLink.tsx

```diff
@@ -0,0 +1,31 @@
+import { LinkIcon, Pill } from "@saleor/macaw-ui";
+import React from "react";
+
+interface AppUrlLinkProps {
+  url: string | null;
+  onClick: () => void;
+}
+
+const AppUrlLink: React.FC<AppUrlLinkProps> = ({ url, onClick }) => {
+  if (typeof url === "string") {
+    return (
+      <Pill
+        icon={<LinkIcon />}
+        label={url as string}
+        color="info"
+        size="small"
+        onClick={onClick}
+      />
```

Comment:
I you want to right-click action it you can try wrapping it with `Link` from `react-router-dom` or setting `component={Link} to={url}`